Update parameter description for Chain-specification.md (#291)
paritytech/wiki: Chain-specification.md

  + `"extraData"` extra data of the genesis block.

+ **`"params"`** contains general chain parameters:
  + `"networkID"` DevP2P supports multiple networks, and ID is used to uniquely identify each one which is used to connect to correct peers and to prevent transaction replay across chains. See [here](https://ethereum.stackexchange.com/questions/17051/how-to-select-a-network-id-or-is-there-a-list-of-network-ids/17101#17101) for network IDs that are not available anymore.

>   + `"maximumExtraDataSize"` integer specifying the maximum size in bytes of the extra_data field of the header. This must be 32 bytes or fewer as per the Ethereum Yellow Paper. It is an optional free space, some Ethereum mining pools use it to logg their blocks. Current value for Ethereum and Kovan is "32".

Reviewer: ```suggestion
  + `"maximumExtraDataSize"` integer specifying the maximum size in bytes of the extra_data field of the header. This must be 32 bytes or fewer as per the Ethereum Yellow Paper. It is an optional value, used by some Ethereum mining pools to log their blocks. The current value for Ethereum and Kovan is "32".
```

  + `"minGasLimit"` the minimum gas limit, should not be smaller than the intrinsic gas used by a transaction which is 21000. Most chains like Ethereum and Kovan, however use 5000 as the initial gas limit, which is too low to send any transactions. This originatied in a deliberate process called "thawing" from the early Ethereum days, where the gas limit started out very low, and was gradually increased over time as people became more confident that the network was functioning correctly.
  
  Optional:
  + `"accountStartNonce"` in the past this was used for transaction replay protection

>   + `"chainID"` chain identifier, was introduced in EIP-155 to prevent replay attacks between the main ETH and ETC chains, which both have a networkID of 1. ChainID is additional way to tell chains apart. Subsequent to EIP-155, ETH has a chainID of 1, while ETC has a chainID of 61 (even though they still have the same networkID of 1). networkID and chainID are  required for the chain to operate in general - e.g. it's required when signing transactions, meaning transactions signed on the ETH network end up with a different hash than those signed on ETC. Before EIP-155, signed transactions on each network would look the same, and could be replayed. See [here](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md) for more information.Default value is equal to `networkID`.

Reviewer: ```suggestion
  + `"chainID"` chain identifier, was introduced in EIP-155 to prevent replay attacks between the main ETH and ETC chains, which both have a networkID of 1. ChainID is an additional way to tell chains apart. Subsequent to EIP-155, ETH had a chainID of 1, while ETC had a chainID of 61 (even though they still have the same networkID of 1). networkID and chainID are  required for the chain to operate in general - e.g. it's required when signing transactions, meaning transactions signed on the ETH network end up with a different hash than those signed on ETC. Before EIP-155, signed transactions on each network looked the same, and could be replayed. See [here](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md) for more information.Default value is equal to `networkID`.
```

>   + `"subprotocolName"` by default its the `eth` subprotocol. All subprotocols names in one place can be found [here]( https://github.com/ethereum/devp2p/issues/47).

Reviewer: ```suggestion
  + `"subprotocolName"` by default its the `eth` subprotocol. All subprotocol names in one place can be found [here]( https://github.com/ethereum/devp2p/issues/47).
```

>   + `"forkBlock"` block number of the latest fork that should be checked by the [Automatic Updating](https://wiki.parity.io/Automatic-Updating) feature.

Reviewer 2: I'd rather speak about the warp-sync than the auto-update.

```suggestion
  + `"forkBlock"` block number of the latest fork that should be verified. In case of chain split, a node would only warp-sync with a peer whose block hash matches the `forkBlockHash` for a given `forkBlock` to make sure to sync with a given side of a fork.
```
